### PR TITLE
Feature/profile creation

### PR DESCRIPTION
**Pull request by imbieras**
Implemented basic user profile creation.

### CODE REVIEW

File: StudyBuddy/Views/ProfileCreation/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@{
+    ViewBag.Title = "title";
```
Comment (canyonhhh) [inline StudyBuddy/Views/ProfileCreation/Index.cshtml:2]: Change to appropriate title

File: StudyBuddy/Models/UserModel.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    public UserId Id { get; }
+
+    public string Name { get; set; }
+
+    public UserFlags Flags { get; set; }
+
+    public UserTraits Traits { get; set; }
+
+    public static string GenerateGravatarUrl(UserId userId) =>
```
Comment (canyonhhh) [inline StudyBuddy/Models/UserModel.cs:25]: Maybe transfer to UserManager?

File: StudyBuddy/Views/ProfileCreation/ProcessNewProfile.cshtml
Comment (canyonhhh) [inline StudyBuddy/Views/ProfileCreation/ProcessNewProfile.cshtml:1]: Refactor to DisplayNewProfile perhaps

**Review by canyonhhh — COMMENTED** [review]
All good haha check comments

File: StudyBuddy/Models/UserModel.cs (hunk 2 above)
Comment (imbieras) [reply StudyBuddy/Models/UserModel.cs:25]: Function has to do with user traits, not managing them.

**Review by canyonhhh — APPROVED** [review]
All G haha

**Pull request merged**